src/CPU.js

```diff
@@ -0,0 +1,58 @@
+/* A class that can serve as a computer-controlled opponent */
+class CPU
+{
+    constructor(identifier, direction, dropablePieces, capturedPieces, color = undefined, isCPU = true)
+	{
+		this.identifier = identifier;
+		this.direction = direction;
+		this.dropablePieces = dropablePieces;
+		this.capturedPieces = capturedPieces;
+		this.color = color;
+        this.isCPU = isCPU;
+        
+        this.caution = 0.75;
```

Comment:
Accept as inputs so these are configurable; include reasonable defaults / extrapolate from inputs in case users do not want to provide all of these.